src/extensions/default/bramble/lib/UI.js

```diff
@@ -111,11 +116,11 @@ define(function (require, exports, module) {
 
         var secondPaneWidth = BrambleStartupState.ui("secondPaneWidth");
         var firstPaneWidth = BrambleStartupState.ui("firstPaneWidth");
-                         
+
```

Comment:
I would be willing to bet that we're not linting this code at all, because it's under `src/extensions/*`.  We should do it at least for the extensions we write.  I'll confirm and file a bug if we aren't.